src/JBArbitrumSucker.sol

```diff
@@ -232,16 +231,23 @@ contract JBArbitrumSucker is JBSucker, IJBArbitrumSucker {
             nativeValue = amount;
         }
 
+        // Ensure we bridge enough for gas costs on L2 side
+        if (transportPayment < feeTotal) revert JBArbitrumSucker_NotEnoughGas(transportPayment, feeTotal);
+
         // Create the retryable ticket containing the merkleRoot.
+        // We call unsafe as we do not want the refund address to be aliased to L2.
```

Comment:
I see that arb devs has put the following @dev tag `@dev Advanced usage only (does not rewrite aliases for excessFeeRefundAddress and callValueRefundAddress).`, but still can't understand why we don't need to aliased the addresses here, but the same are being overridden in `createRetryableTicket`, while under the hood the logic is the same after that. Did find something about that?

Reply:
We want the sucker on ARB L2 to receive the funds, but because the L1 and L2 sucker are both at the same address we need it to not alias the address. As the aliased address is only accessible by the L1 sucker.